README.md

```diff
@@ -45,6 +45,11 @@ When you become familiar to the TizenRT build system, you may use the *dbuild.sh
 As the build script is running based on Docker, it requires *sudo* for root permission.  
 To run Docker without *sudo*, refer to https://docs.docker.com/install/linux/linux-postinstall/.
 
+If your OS is Windows, you need to configure git like below before git clone:
+```
```

Comment:
For windows environment, configuring CRLF is required as below.

Reply:
Fixed!